presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -749,3 +749,14 @@ Hive 3 Related Limitations
   not correctly read ``timestamp`` values from Parquet, RCBinary, or Avro
   file formats created by Hive 3.1 or later. When reading from these file formats,
   Presto returns different results than Hive.
+* :doc:`/sql/create-table-as` can be used to create transactional tables in ORC format like this::
```

Comment:
Add newlines between all of the list items for readability (now that we have a list item with blank lines)